production_ready.md

```diff
@@ -0,0 +1,25 @@
+# How GrandRounds defines "Production Ready"
+
+To launch a service in production your service must have the following:
+- [ ] Automated configuration and deployment
+- [ ] Security review and access controls in place
+- [ ] Monitoring configured and verified. Alerting setup
+- [ ] Scaling plan and Availablity plan defined
+- [ ] Backup and Recovery plans exist and have been tested
+
+## Automated configuration and deployment
+Service/system should be defined through configuration files, base images, setup scripts. There should be no manual configuration or setup of system. Instructions should be available along with any setup scripts. Instructions should be detailed enough to hand over to someone else to run. The system should be able to be created in the common environments we use (production, UAT, integration, CI, development) using automated tooling. The automation scripts serve as 
+
+## Security review and access controls
+Submit your review of the security of the service. The review should include what attack surfaces are created by this service and how you mitigate them. Detail who/what systems access this new service and how. Define the service’s ACLs and for the deployment of the system. Describe the types of security and encryption used. Include details of the PHI/PII types. Re-encryption plan/schedule is needed as well.
+
+## Monitoring/Alerting configured and verified
+Set up your monitoring system for the service in each environment. This should include status health checks for the ELB (if applicable), system level metrics, service logs, reporting metrics and alerting thresholds.
+
+## Scaling plan and Availability plan
```

Comment:
Well, I'm  not hung up on 5 vs. 6, and I think they're both important but in different ways.  At the moment, almost *none* of our stuff needs to scale beyond one instance (excepting jarvis), yet for availability reasons, we have a minimum deployment count of 2.  Also, while availability is important at almost every scale, *scale itself* isn't always a goal of a service, and *arbitrary* scale is not always a goal worth fighting for.  E.g., unless our business pivots dramatically, there is no point in us talking about 1000 rps scale, yet it's sexy and folks will throw it up as a target.  That's just not our bird.

(Hmm, thought I sent this yesterday, sorry :()